_includes/changelog.html

```diff
@@ -1,10 +1,10 @@
-{% assign componentName = page.changelog.key | default: page.title | downcase %}
-{% assign componentKey = componentName | slugify %}
-{% assign changelog = site.data.changelogs[componentKey] %}
+{% assign pageKey = page.changelog.key | default: page.title | downcase %}
+{% assign pageSlug = pageKey | slugify %}
+{% assign changelog = site.data.changelogs[pageSlug] %}
 {% assign changelogItems = changelog.items %}
 
 {%- if changelog.items or changelog.changelogURL -%}
-  <h2 id="{{ componentKey }}-changelog" class="site-component-section-title">Latest updates</h2>
```

Comment:
Updated the anchor link to just point to `#changelog`. Using the pageKey didn't feel necessary and would require updates if page titles/keys change.